server/src/com/cloud/storage/VolumeApiServiceImpl.java

```diff
@@ -403,8 +415,11 @@ private boolean validateVolume(Account caller, long ownerId, Long zoneId, String
                 throw new InvalidParameterValueException("File:// type urls are currently unsupported");
             }
             UriUtils.validateUrl(format, url);
-        // check URL existence
-        UriUtils.checkUrlExistence(url);
+
+            // check URL existence
+            if(ValidateURLExistence.value()) {
```

Comment:
I see some similar changes in https://github.com/apache/cloudstack/pull/2074 as well